owl_wms/nn/rope.py

```diff
@@ -11,103 +11,66 @@
 from torch import nn
 import time
 
+
 class FlatVideoRoPE(nn.Module):
```

Comment:
Need to make sure this works the same way once shahbuland provides you checkpoints.